models/gitm/README.rst

```diff
@@ -0,0 +1,335 @@
+###########
+GITM README
+###########
+
+`NAMELIST <#Namelist>`__ / `INTERFACES <#Interface>`__ / `FILES <#FilesUsed>`__
+/ `REFERENCES <#References>`__ / `ERRORS <#Errors>`__ / `PLANS <#FuturePlans>`__
+/ `PRIVATE COMPONENTS <#PrivateComponents>`__ / `TERMS OF USE <#Legalese>`__
+
+Contents
+========
+
+#. `Overview`_
+#. `Compiling`_
+#. `Converting Between DART Files and GITM Restart Files`_
+#. `Simple Test`_
+#. `Namelist`_
+#. `Files`_
+#. `Terms of Use`_
+#. `References`_
+
+Overview
+========
+
+The `Global Ionosphere Thermosphere Model
+(GITM) <http://ccmc.gsfc.nasa.gov/models/modelinfo.php?model=GITM>`__ is a
+3-dimensional spherical code that models the Earth's thermosphere and ionosphere
+system using a stretched grid in latitude and altitude.
```

Comment:
You have no way of knowing this, but the version in this directory is for LANAI.  Please add the following statement:

> The Manhattan support for WP-GITM is being developed. Persons interested in using any version of GITM with the Manhattan version of DART are encouraged to contact the DART development team.

And if you could make that BLINKING in 100-point font, I'd appreciate it.

Reply:
This is the project that we are working on with JPL. I don't really know if we are allowed to merge that branch onto master yet.